somoye123/Enumerables-Ruby
# Enumerable Methods

### This project implements a copy of some Enumerable methods in Ruby
        This project was built to solidify knowledge of Ruby.


#### HOW TO TEST
- clone this [repo](https://github.com/somoye123/Enumerables-Ruby)
- cd in into the cloned repo
-  Scroll down to the bottom of the page and uncomment a test case of your choice or add your test.
- Open a terminal and run `ruby enumerable.rb`.
- Verify the output in the terminal  against the expected result of the given test case
